src/block.rs

```diff
@@ -4,27 +4,29 @@ type BlockAddr = u64;
 /// Information about a basic block.
 #[derive(Debug, Eq, PartialEq)]
 pub struct Block {
-    /// Virtual address of the first instruction in this block.
+    /// Virtual address of the start of the first instruction in this block.
```

Comment:
It wasn't obvious to me if it pointed to the start or end of the last instruction.

Reply:
Hrm. But why would the address of an instruction be defined as "it's last byte"? That'd be super odd.

Reply:
This is basically a contiguous vector of things, right? There are two common ways of encoding this: start address & length; first byte & last byte. What we've got with `Block` at the moment is the far less common "start address & start address of last element". Because it's unusual I think it needs documenting.

Reply:
I'm not gonna die on this hill, so I'll allow it :P